lls/main.c

```diff
@@ -200,13 +206,14 @@ put_nd(struct in6_addr *ip_be, unsigned int lcore_id)
 }
 
 int
-submit_nd(struct rte_mbuf *pkt, struct gatekeeper_if *iface)
+submit_nd(struct rte_mbuf **pkts, int num_pkts, struct gatekeeper_if *iface)
 {
 	if (nd_enabled(&lls_conf)) {
```

Comment:
Shouldn't this test be for ND at `iface` instead of ND in general? Even deeper, is this test needed at all since the ACLs are only installed if ND is enable at that interface?